File: eAutokuca/eAutokuca.Models/Recenzije.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eAutokuca.Models
{
    public class Recenzije
    {
        public int RecenzijeId { get; set; }

        public string Sadrzaj { get; set; } = null!;

        public int? KorisnikId { get; set; }

        public int Ocjena { get; set; }

        public virtual Korisnik? Korisnik { get; set; }
    }
}

File: eAutokuca/UI/eautokuca_mobile/lib/models/recenzije.dart
import 'package:eautokuca_mobile/models/korisnici.dart';
import 'package:json_annotation/json_annotation.dart';

part 'recenzije.g.dart';

@JsonSerializable()
class Recenzije {
  int? recenzijeId;
  String? sadrzaj;
  int? ocjena;
  Korisnici? korisnik;

  Recenzije(
    this.recenzijeId,
    this.sadrzaj,
    this.ocjena,
    this.korisnik,
  );

  factory Recenzije.fromJson(Map<String, dynamic> json) =>
      _$RecenzijeFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$RecenzijeToJson(this);
}
